eos-extra-target-generator

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+
+normal_dir="$1"
+target_unit="${1}/eos-extra-settled.target"
+
+cat >"${target_unit}" <<EOF
+[Unit]
+Description=endless-extra SD card mounted (if expected)
+Wants=local-fs.target
+EOF
+
+if [ -L "/etc/systemd/system/local-fs.target.wants/var-endless\x2dextra.mount" ]; then
```

Comment:
Rather than testing this path, I think it would be better to use `systemctl is-enabled --quiet "var-endless\x2dextra.mount"` so that it doesn't depend on how the image builder enables the unit. The only thing I'd be worried about is somehow getting systemd into a deadlock doing that.

Reply:
According to the manpage we can't use `systemctl` from generators:
```
Generators are run very early at boot and cannot rely on any external services. They may not talk to any other process. That includes simple things such as logging to syslog(3), or systemd itself (this means: no systemctl(1))!
```

Reply:
@jprvita thanks for checking! Let's just keep the simple check then. That said, it would be more correct to just check for existence of the file rather than whether it's a symlink. There's no reason it has to be a symlink besides convenience. Meh, this is fine as is.